CONTRIBUTING.md

```diff
@@ -1,45 +1,79 @@
-How the code in this repo works
-===
+#How the code in this repo works
+
 Hi! Glad you want to add code to this project. First, a brief overview of what's gone into this repo and some suggestions on how to get started using it.
 
-The frontend stack
----
+Formally, this repository contains a Redux app written with ES6 JavaScript, styled with Sass, and served with hotloading and module support from Webpack. It relies on a backend with a RESTful API, which is a Django server running PostgreSQL in Docker. 
+
+The section "Uh, where does the code even start?" gives a brief overview. The two parts after that detail the frontend and backend stacks, respectively.
+
+#Uh, where does the code even start?
+
+The main body of code for the frontend is in the `app` folder. In the `app` folder, `index.js` adds `Router`, the root React component. `Router` uses `routes.js` to decide what component to render for a given route/url. On the initial load, looking at `routes.js` shows that the `App` component from `app.js` is loaded for the route `/`, or at `http://localhost:3001/app/#/`. 
```

Comment:
Can someone who is not as familiar with some of this stuff speak to how useful this is?